pkg/cron/cron.go

```diff
@@ -96,7 +96,7 @@ func notifyOnErr(err error, backoffDelay time.Duration) {
 // RequestCall - ZMS call for update crons
 func (c *Cron) requestCall() error {
 	master := false
-	domains, etag, err := c.zmsClient.GetSignedDomains("", "true", "", &master, c.etag)
+	domains, etag, err := c.zmsClient.GetSignedDomains("", "true", "", &master,&master, c.etag)
```

Comment:
Would you give more information regarding this change?

Reply:
Hi @zetaj ,
Please find the attached screenshot of the function `GetSignedDomains()`.
The `conditions` field has been added as a parameter to the updated package. So to accomodate this, we are sending it as a boolean set to `false`.

![Screenshot 2024-06-03 at 10 17 01 AM](https://github.com/AthenZ/k8s-athenz-syncer/assets/65297307/05da7fd1-2916-49a1-8936-1e5c607b480a)